### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by whoami15**

### CODE REVIEW

File: app/Http/Controllers/TasksController.php [hunk 1]
```diff
@@ -67,65 +67,45 @@ public function show($id)
...
      */
-    public function edit($id)
+    public function edit(Task $task)
     {
-        $task = Task::find($id);
-
         if (empty($task)) return response(['message' => 'Task not found'], 404);
 
-        return view('templates.edit-task', ['task' => $task]);
+        return view('templates.edit-task', $task);
```
Comment (leticiadia) [inline app/Http/Controllers/TasksController.php]: Just a small problem found in that part of the code. When trying to edit the task, it returned undefined $task, so I needed to add the method compact('task')  passing task and it worked.

would be like this: return view('templates.edit-task', compact('task'));

**Review by leticiadia — CHANGES_REQUESTED** [review]
Adjust in edit method.

File: app/Http/Controllers/TasksController.php (hunk 1 above)
Comment (whoami15) [reply app/Http/Controllers/TasksController.php]: Oh sorry, will update it.

**Review by leticiadia — APPROVED** [review]
Approved pull request!

**Pull request merged**